modules/claims_api/spec/support/rswag_config.rb

```diff
@@ -6,8 +6,9 @@ def config # rubocop:disable Metrics/MethodLength
       'modules/claims_api/app/swagger/claims_api/v1/swagger.json' => {
         openapi: '3.0.1',
         info: {
-          title: 'API V1',
+          title: 'Benefits Claims',
```

Comment:
I noticed this is something that Appeals includes in their [rwag config](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/spec/support/rswag_config.rb#L16). Figured it doesn't hurt to add them to ours, as well